app/src/main/java/com/jerboa/ui/components/post/PostListings.kt

```diff
@@ -73,32 +74,39 @@ fun PostListings(
                 postView.post.id
             },
         ) { postView ->
-            PostListing(
-                postView = postView,
-                onUpvoteClick = onUpvoteClick,
-                onDownvoteClick = onDownvoteClick,
-                onPostClick = onPostClick,
-                onSaveClick = onSaveClick,
-                onCommunityClick = onCommunityClick,
-                onEditPostClick = onEditPostClick,
-                onDeletePostClick = onDeletePostClick,
-                onReportClick = onReportClick,
-                onPersonClick = onPersonClick,
-                onBlockCommunityClick = onBlockCommunityClick,
-                onBlockCreatorClick = onBlockCreatorClick,
-                isModerator = false,
-                showCommunityName = showCommunityName,
-                fullBody = false,
-                account = account, // TODO can't know with many posts
-                postViewMode = postViewMode,
-                showVotingArrowsInListView = showVotingArrowsInListView,
-                enableDownVotes = enableDownVotes,
-                showAvatar = showAvatar,
-                useCustomTabs = useCustomTabs,
-                usePrivateTabs = usePrivateTabs,
-                blurNSFW = blurNSFW,
-            )
-            Divider(modifier = Modifier.padding(bottom = SMALL_PADDING))
+            Card(
+                colors = CardDefaults.cardColors(
+                    containerColor = MaterialTheme.colorScheme.background,
+                ),
+                modifier = Modifier
+                    .padding(12.dp),
```

Comment:
Thoughts on something closer to `SMALL_PADDING` here?

Reply:
I was afraid the effect would be too subtle with `SMALL_PADDING`, but it still works. I've removed the bottom padding entirely to ensure the amount of whitespace between cards equals `SMALL_PADDING`.